workflow/rules/slice.smk

```diff
@@ -1,7 +1,7 @@
 # Use osmium to cut a .osm.pbf file into several .osm.pbf files as defined by bounding boxes
 rule slice:
     input:
-        # data=os.path.join(DATA_DIR, f"{DATASET}.osm.pbf"),
+        # data=os.path.join(config['data_dir'], f"{DATASET}.osm.pbf"),
```

Comment:
The workflow update `rule slice` has been modified to be snakemakeified. We'll follow that kind of lead where we can in this PR.